components/table/index.css

```diff
@@ -12,511 +12,348 @@
  */
 
 .spectrum-Table {
```

Comment:
A lot of the CSS has just been reorganized to more or less match what we are doing in other components. There was a lot of consolidating of the custom properties up at the top that needed to happen.